Stable version

igoradamenko opened the pull request.

igoradamenko committed:
init commit

igoradamenko committed:
global refactoring

igoradamenko committed:
geek news handler fix

igoradamenko committed:
add author to info

igoradamenko committed:
add async to GETs

igoradamenko committed:
some visual enhancements for mobiles

igoradamenko committed:
fix auth popup prevention

igoradamenko committed:
add bookmark for quick add page to news

igoradamenko committed:
fix auth popup prevention in safari

God bless me

igoradamenko committed:
fix logout

it's last fix, I'm sure

igoradamenko committed:
fix date formatting

igoradamenko committed:
add rss-link to head

igoradamenko committed:
add favicon

igoradamenko committed:
add full post + disqus comments; refactoring

igoradamenko committed:
fix post link in admin panel

igoradamenko committed:
fix back link for admins; visual fixes

igoradamenko committed:
add support GET login & password

igoradamenko committed:
remove unusable auth by GET

igoradamenko committed:
improve quick-add-bookmark

igoradamenko committed:
fix logout

igoradamenko committed:
remove repeatable code

igoradamenko committed:
increase zindex of bookmark

igoradamenko committed:
fix disqus comments counter when value == 0

igoradamenko committed:
change fonts urls to https

igoradamenko committed:
change apipath to https

igoradamenko committed:
add disqus id for counters

igoradamenko committed:
Revert "add disqus id for counters"

This reverts commit cedb9c73837566e4c5f136d360b538ca681e32fa.

igoradamenko committed:
fix min-height of news items

igoradamenko committed:
add auto hide visible news content

igoradamenko committed:
fix content styles for unauthorized users

igoradamenko committed:
add mobile check for auto hide news

igoradamenko committed:
add long pooling for news

igoradamenko committed:
add test files to ignore

igoradamenko committed:
improve sortable news

igoradamenko committed:
fix all/geek switcher

snater commented on src/components/Main/Main.js:
```diff
@@ -9,17 +10,26 @@ export default class Main extends React.Component {
         this.store.onChange(this.onStoreChange.bind(this));
 
         this.state = this.store.getState();
+        this.componentDidMount()
```
Это специальный метод у реакта, он автоматом вызывается

snater commented on src/helpers/request.js:
```diff
@@ -0,0 +1,6 @@
+import xhttp from 'xhttp/native';
```
Ах ты читер)

olegpopadko replied:
надо крутиться)

igoradamenko committed:
add source link to post

igoradamenko committed:
fix comments link + add 'to-comment' link

igoradamenko committed:
add animation for 'to-comment' link

igoradamenko committed:
fix to-comment link show

before: after json2dom
now: when `news-loaded` event triggered

igoradamenko committed:
load disqus BEFORE json2dom

igoradamenko committed:
Revert "load disqus BEFORE json2dom"

This reverts commit 858fc9a7817eed9001cd57d33b2f0e2c937c471d.

igoradamenko committed:
add auto redirect from /login

igoradamenko committed:
increase scroll sensitivity for sortable news

igoradamenko committed:
fix #1

+ add auto hide news when open another (for mobile)

igoradamenko committed:
replace disqus jquery code to vanilla js

igoradamenko committed:
fix disqus count.js creating

igoradamenko committed:
remove animation of toggling news

igoradamenko committed:
add 304 error handler for updateCurrent

igoradamenko committed:
edit links to source, comments & slug

+ add interval checking of disqus loading

igoradamenko committed:
add google analytics code

igoradamenko committed:
add day check for news long polling

igoradamenko committed:
add always update current for admins

igoradamenko committed:
add geek indicator for users

igoradamenko committed:
add geek indicator for users to one news

igoradamenko committed:
replace disqus count.js by server data

igoradamenko committed:
add news sorting; add visual enhancements for dels

igoradamenko committed:
fix geek news click when → /#geek

igoradamenko committed:
add save state when change all/geek news

igoradamenko committed:
add autoload full news when click 'more'

+ remove full news from news list

igoradamenko committed:
add handler for sotrable news list

igoradamenko committed:
add cursor for sort-handler

igoradamenko committed:
create disable/enable news sortable funcs

igoradamenko committed:
fix enabling of news sorting

igoradamenko committed:
fix to-current link

igoradamenko committed:
add test version of new quick-add

igoradamenko committed:
improve quick add bookmarklet

igoradamenko committed:
spelling fix

igoradamenko committed:
remove old bookmarklet

igoradamenko committed:
docker test

igoradamenko committed:
add likes & sort by them

igoradamenko committed:
improve likes & comments links

igoradamenko committed:
fix likes & comments & color them

igoradamenko committed:
update docker files

igoradamenko committed:
add volume section

igoradamenko committed:
change comments & likes icons color

igoradamenko committed:
move likes & comments + add hyphens

igoradamenko committed:
add archive; refactor main menu

igoradamenko committed:
remove priority sorting from archive

igoradamenko committed:
fix default active sorting for archive

igoradamenko committed:
fix autoscrolling to comments

igoradamenko committed:
fix sortable

igoradamenko committed:
add auto redirect for admins from / to /admin/

igoradamenko committed:
improve working with svg icons

add new packages for generate svg sprites and inject them to html; all
`content` props for svg changed to inline svg using `use`

igoradamenko committed:
fix button `сделать первой` hiding

igoradamenko committed:
add page for list of rules

igoradamenko committed:
add edit page for rules

igoradamenko committed:
add page for add rule + some fixes

igoradamenko committed:
remove the first test file of rep, lol

igoradamenko committed:
Revert "add page for add rule + some fixes"

This reverts commit bc386d8fb71cfdb06d2d39c4ee7b69a3cc845868.

igoradamenko committed:
Revert "add edit page for rules"

This reverts commit 7a10cbd8ca1fb99063be3fdb48d6982738591fc3.

igoradamenko committed:
Revert "add page for list of rules"

This reverts commit 5acc926c2638f97b93b507dfe768db4d04d1ce25.

igoradamenko committed:
fix reverting commits

igoradamenko committed:
add positions check

igoradamenko committed:
add func for reset positions + reset on delete

igoradamenko committed:
refactor sorting links

igoradamenko committed:
add filters + a little bit refactor

igoradamenko committed:
fix filters :)

igoradamenko committed:
remove link 'Гиковские' from header...

+ add switch off news sortable when filters are turned on

igoradamenko committed:
change recent time for geeks to 2 m + w

igoradamenko committed:
add dropdown for filter + common filter

igoradamenko committed:
change view of filter's dropdown

igoradamenko committed:
add show start link

igoradamenko committed:
Merge remote-tracking branch 'refs/remotes/umputun/master'

igoradamenko committed:
move show start link to footer

igoradamenko committed:
add delay for updateCurrent if it fails

igoradamenko committed:
add scroll top for show start button

igoradamenko committed:
hide images, etc in news desc

igoradamenko committed:
add switchers func & markup

igoradamenko committed:
add markup for manual addition

igoradamenko committed:
minify output css & js

igoradamenko committed:
add font-size for top status

igoradamenko committed:
add fold-up panel for full news in list

igoradamenko committed:
add manual news addition

igoradamenko committed:
change tasks order

igoradamenko committed:
fix merge conflicts

igoradamenko committed:
merge

igoradamenko committed:
fix merging

igoradamenko committed:
yet another merge fix

igoradamenko committed:
Merge pull request #2 from igoradamenko/featuring

Merge
